### PR TITLE
Ra 2359

### PR DESCRIPTION
**Pull request by fawques**

### CODE REVIEW

File: LCU.NET.Extras/Game/Actuator.Game.cs [hunk 1]
```diff
@@ -306,5 +306,18 @@ public async Task UploadSpells (Position pos, int championId)
 				LogTo.Debug ("Uploaded spells");
 			}
 		}
+
+		public async Task<bool> CanConnect ()
+		{
+			bool allowed = false;
+			try {
+				string platform = await LeagueClient.MakeRequestAsync<string> ("/lol-platform-config/v1/namespaces/LoginDataPacket/platformId", Method.GET);
+				allowed = platform != "KR";
```
Comment (ebargues) [inline LCU.NET.Extras/Game/Actuator.Game.cs]: Can we put "KR" in some constants class?

**Review by ebargues — COMMENTED** [review]
Is there any reason for not doing tests here?

File: LCU.NET.Extras/Game/Actuator.Game.cs [hunk 2]
```diff
@@ -203,29 +203,23 @@ public async Task UploadItemSet (Position pos, int championId)
...
-					}
-					LogTo.Debug ("Downloaded from provider");
-				} else {
-					Main.ShowNotification (string.Format ("Rune page for champion {0} not set", championId), null, NotificationType.Error);
-					return;
-				}
+
+				var positionName = pos.ToString ().ToLower ();
+				LogTo.Debug ($"Could not retrieve a valid rune page for {championId} in {positionName}");
+				Main.ShowNotification ($"No runes for {positionName}", $"There's not enough info to set runes for {championId} in {positionName}", NotificationType.Error);
```
Comment (axelgorris) [inline LCU.NET.Extras/Game/Actuator.Game.cs]: Main.ShowNotification ($"No runes for position {positionName}", $"There's not enough info to set runes for your champion in position {positionName}", NotificationType.Error);

**Comment by fawques** [comment]
> 
> 
> Is there any reason for not doing tests here?

Not really, just that it's not our code, and there were no tests to begin with

File: LCU.NET.Extras/Game/Actuator.Game.cs (hunk 1 above)
Comment (fawques) [reply LCU.NET.Extras/Game/Actuator.Game.cs]: 👌

**Review by ebargues — APPROVED** [review]

**Pull request merged**